packages/cli/src/types.ts

```diff
@@ -199,3 +197,82 @@ export interface GenerateDeploymentChainInterface {
   flags: DeploymentFlagsInterface;
 }
 
+export interface SubgraphTemplateSource {
```

Comment:
I think these can live in a separate file, lets keep subgraph types separate to ours where possible